web/samples/webpack/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+'use strict';
+
+module.exports = {
+    devtool: 'source-map',
+    entry: './src/app.ts',
+    externals: {
+        fs: 'fs',
+        crypto: 'crypto',
```

Comment:
2 `crypto`? Also, `crypto`? I assume it has to do with webpack.

Reply:
Yikes good catch, also added explanatory comment